content/_data/changelogs/lts.yml

```diff
@@ -3149,6 +3149,265 @@
       Revert Winstone from 5.8 to 5.3 to resolve embedded Jetty web container regressions in later Winstone versions.
       High default maximum form size limit and reverse proxy redirection are restored (regressions in 2.204.3).
 
+- version: 2.222.1
+  date: 2020-03-25
+  banner: >
+    <b>Note:</b> important text will be inserted here.
+  lts_predecessor: 2.204.5
+  lts_baseline: '2.222'
+  changes: # compared to lts_baseline 2.222
+  - type: major bug
+    category: regression
+    pull: 4568
+    issue: 61429
+    authors:
+    - fqueiruga
+    message: |-
+      Fix drag & drop for previously saved steps in the job configuration form (regression in 2.217).
+  - type: major bug
+    category: regression
+    pull: 4542
+    issue: 60409
+    references:
+      - pull: 4542
+      - issue: 60409
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.9
+        title: Winstone 5.9 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.27.v20200227
+        title: Jetty 9.4.27 changelog
+    authors:
+    - jglick
+    - olamy
+    message: |-
+      Winstone 5.9: Fix propagation of the maximum form content size and form content keys number (regression in Jetty 9.4.20 and Jenkins 2.205).
+  - type: major bug
+    category: regression
+    pull: 4542
+    issue: 60199
+    authors:
+    - jglick
+    - olamy
+    references:
+      - pull: 4542
+      - issue: 60199
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.9
+        title: Winstone 5.9 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.27.v20200227
+        title: Jetty 9.4.27 changelog
+    message: |-
+      Winstone 5.9: Fix improper reverse proxy redirects to host due to X-Forwarded-Host and X-Forwarded-Port ordering issue (regression in Jetty 9.4.20 and Jenkins 2.205).
+  - type: bug
+    category: regression
+    pull: 4452
+    issue: 57888
+    authors:
+    - lrpg
+    message: |-
+      Winstone 5.9: Fix support of system logging customization (regression in 2.177)
+    references:
+      - pull: 4452
+      - issue: 57888
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.9
+        title: Winstone 5.9 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.27.v20200227
+        title: Jetty 9.4.27 changelog
```

Comment:
It was fixed in 5.7, and hence 2.222 is not impacted . https://github.com/jenkinsci/winstone/releases/tag/winstone-5.7
I believe 

```suggestion
```

Reply:
I had inserted it here because 2.204.5 intentionally regressed the customized logging functionality in favor of other fixes.  I'm also fine removing it from the changelog.

Reply:
Hence this should be in the other section, which covers changes since 2.204.5.

Reply:
Thanks for the clarity.  I've moved it into the 2.204.5 section with 2982f5c3227ab2aa1ecec7b985f023bbe2364c0b

Reply:
Screenshot updated and upgrade guide added.